File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){

    int height;

    do {
        height = get_int("Height: ");
    } while (height <= 0 && height > 25);

    char hash = '#';
    char space = ' ';
    int current = 2;

    for(int i=0; i<height; i++){
        for(int j=0;j<(height-current+1);j++){
            printf("%c", space);
        }
        for(int k=0; k<current;k++){
            printf("%c", hash);
        }
        printf("\n");
        current++;
    }
}

File: pset6/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 0 and height < 25:
        break


hash = '#';
space = ' ';
current = 2;

for i in range(height):
    for j in range(height - current + 1):
        print(space, end="")
    for k in range(current):
        print(hash, end="")
    print()
    current += 1